core/lib/workarea/configuration/redis.rb

```diff
@@ -39,11 +39,15 @@ def find_config(name)
           return from_config if from_config.present?
 
           env_slug = name.to_s.underscore.upcase
+          scheme = ENV["WORKAREA_#{env_slug}_SCHEME"].presence || DEFAULT[:scheme]
 
           {
+            scheme: scheme,
             host: ENV["WORKAREA_#{env_slug}_HOST"].presence || DEFAULT[:host],
             port: ENV["WORKAREA_#{env_slug}_PORT"].presence || DEFAULT[:port],
-            db: ENV["WORKAREA_#{env_slug}_DB"].presence || DEFAULT[:db]
+            db: ENV["WORKAREA_#{env_slug}_DB"].presence || DEFAULT[:db],
+            password: ENV["WORKAREA_#{env_slug}_PASSWORD"].presence,
+            ssl: scheme == 'rediss' ? true : false
```

Comment:
```suggestion
            ssl: scheme == 'redis' ? true : false
```

Reply:
@MMartyn I can't commit this suggestion since its your personal fork, can you commit that?

Reply:
I can see how it might seem like a typo, but rediss was intentional.